frontend/src/components/dashboard/ProfileBanners.tsx

```diff
@@ -194,26 +201,55 @@ const NoChromeExtensionBanner = () => {
 type NoPremiumBannerProps = {
   runtimeData: RuntimeDataWithPremiumAvailable;
 };
-const NoPremiumBanner = (props: NoPremiumBannerProps) => {
+
+// const NoPremiumBanner = (props: NoPremiumBannerProps) => {
+//   const { l10n } = useLocalization();
+
+//   return (
+//     <Banner
+//       key="premium-banner"
+//       type="promo"
+//       title={l10n.getString("banner-upgrade-headline")}
+//       illustration={<img src={RelayLogo.src} alt="" width={60} height={60} />}
+//       cta={{
+//         target: getPremiumSubscribeLink(props.runtimeData),
+//         content: l10n.getString("banner-upgrade-cta"),
+//         onClick: () => trackPurchaseStart(),
+//         gaViewPing: {
+//           category: "Purchase Button",
+//           label: "profile-banner-promo",
+//         },
+//       }}
+//     >
+//       <p>{l10n.getString("banner-upgrade-copy-2")}</p>
+//     </Banner>
+//   );
+// };
+
+const LoyalistPremiumBanner = (props: NoPremiumBannerProps) => {
   const { l10n } = useLocalization();
 
   return (
     <Banner
       key="premium-banner"
       type="promo"
-      title={l10n.getString("banner-upgrade-headline")}
-      illustration={<img src={RelayLogo.src} alt="" width={60} height={60} />}
-      cta={{
+      title={l10n.getString("banner-upgrade-loyalist-headline")}
+      illustration={<img src={FirefoxLogo.src} alt="" width={60} height={60} />}
+      ctaLargeButton={{
         target: getPremiumSubscribeLink(props.runtimeData),
-        content: l10n.getString("banner-upgrade-cta"),
+        content: l10n.getString("banner-upgrade-loyalist-cta"),
         onClick: () => trackPurchaseStart(),
         gaViewPing: {
           category: "Purchase Button",
-          label: "profile-banner-promo",
+          label: "profile-banner-loyalist-promo",
```

Comment:
I've only updated the GA label to track the performance of this banner as opposed to the premium banner - is there anywhere else I should change from the old banner build? @Vinnl

Reply:
Nope, that's it!